zeebe/snapshot/src/main/java/io/camunda/zeebe/snapshots/PersistedSnapshot.java

```diff
@@ -100,6 +105,26 @@ public interface PersistedSnapshot {
    */
   ActorFuture<SnapshotReservation> reserve();
 
+  default boolean isBootstrap() {
+    return getMetadata() != null && getMetadata().isBootstrap();
+  }
+
   @VisibleForTesting
   boolean isReserved();
+
+  default Map<String, Path> files() {
+    final var map = new HashMap<String, Path>();
```

Comment:
:question: Is this safe to do without going through the actor? What if my persisted snapshot is deleted at the same time? :thinking: